packages/webpack/lib/shims/develop.js

```diff
@@ -1,5 +1,13 @@
 /* eslint-env browser */
 
+if (process.env.__HOPS_FAST_BUILD__ === 'true') {
+  if (process.env.__HOPS_FAST_DEV__ !== 'true') {
```

Comment:
Can you add a comment here, why we need to disable this with fast-dev?

Reply:
Thats probably, because we do not polyfill at all with fast-dev right?
But I think its not obvious in this case

Reply:
Yes exactly, I can also leave a comment in code 👍